src/scores/utils.py

```diff
@@ -216,3 +217,79 @@ def check_dims(xr_data: XarrayLike, expected_dims: list[str], mode: Optional[str
                     f"Dimensions {list(xr_data[data_var].dims)} of data variable "
                     f"'{data_var}' are not {mode} to the dimensions {sorted(dims_set)}"
                 )
+
+
+def docstring_param(*args, **kwargs):
```

Comment:
I'm not sure whether this is worth the complication. I'm open to being convinced but I'm not sure there's enough reason for this.

Reply:
Its currently only being used in two places I think it'll come in handy but I'm only using it in 2 places so its [probably not required for now](https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it).